Client/src/components/contanier/staff/Student/AddStudent/index.js

```diff
@@ -0,0 +1,124 @@
+          return acc;
+        }, {});
+        this.setState({ Error: { ...errors } })
+      })
+  }
+  render() {
+    const { firstname, address, level, phonenumber, lastname, mobilenumber, password, Error } = this.state;
+    return (
+      <div className='add-student'>
+        <h1 className='add-student-title'>
+          <span className='add-student-title-border'>
+            Add  Student
+          </span>
+        </h1>
+        <form onSubmit={(e) => this.handleClick(e)} className='add-student-contanier'>
+          <div className='add-student-contanier-left'>
+            <LabeledInput
+              labelText='First name'
+              id='firstname'
+              name='firstname'
+              value={firstname}
+              placeholder='First name '
+              onChange={this.handleInput}
+              Error={Error['firstname']}
+            />
+            <LabeledInput
+              labelText='Address'
+              id='address'
+              name='address'
+              value={address}
+              placeholder='Address'
+              onChange={this.handleInput}
+              Error={Error['address']}
+            />
+            <LabeledInput
+              labelText='Level'
+              id='level'
+              name='level'
+              value={level}
+              placeholder='Level'
+              onChange={this.handleInput}
+              type='Number'
+              Error={Error['level']}
+            />
+            <LabeledInput
+              labelText='Phone number '
+              id='phonenumber'
+              name='phonenumber'
+              value={phonenumber}
+              placeholder='Phone Number '
+              onChange={this.handleInput}
+              Error={Error['phonenumber']}
+            />
+          </div>
+          <div className='add-student-contanier-center'></div>
+          <div className='add-student-contanier-right'>
+            <LabeledInput
+              labelText='Last name'
+              id='lastname'
+              name='lastname'
+              value={lastname}
+              placeholder='Last name '
+              onChange={this.handleInput}
+              Error={Error['lastname']}
+            />
+            <LabeledInput
+              labelText='Mobile number'
+              id='mobilenumber'
+              name='mobilenumber'
+              value={mobilenumber}
+              placeholder='Mobile number'
+              onChange={this.handleInput}
+              Error={Error['mobilenumber']}
+            />
+            <LabeledInput
+              labelText='Password'
+              type='password'
+              id='password'
+              value={password}
+              name='password'
+              placeholder='Password'
+              onChange={this.handleInput}
+              Error={Error['password']}
+            />
+            <Button type='submit' content='Add' className='add-student-btn' onClick={(e) => this.handleClick(e)} />
```

Comment:
@Fatmasiam You can remove the `() => ` and just use handleClick directly, so like `onClick={this.handleClick}`.  This is better for performance, because whenever you have a `() => ..` inside render() like that, that little function is recreated every time the component renders (so it can be a lot of unnecessary copies of that function in memory).